package.json

```diff
@@ -77,6 +77,7 @@
     "metro-symbolicate": "^0.66.2",
     "metro-transform-plugins": "^0.66.2",
     "metro-transform-worker": "^0.66.2",
+    "plist": "^3.0.5",
```

Comment:
What's this for?

Reply:
The latest version of Appium pulls in plist 3.0.4 which has a security issue. Since their dependency syntax specifically asks for 3.0.4, the package with the security issue will be pulled in even though there's a version with the fix out. This line just says "if someone asks for plist force them to resolve to version ^3.0.5

Reply:
This is on top of the need for simple-plist? Dang..

Reply:
Just wondering - is there a reason why this is listed in the root package.json versus the package.json under apps/win32 and apps/windows? Does "resolutions" only work in the root package.json?

Reply:
Nothing in the docs says that it won't work outside of the root package.json, but my guess was that it was put here since then you only have to specify it once to have it apply to the other packages.